EncounterMe/EncounterMe/Views/Popups/AddByAdressPopup.xaml.cs

```diff
@@ -26,8 +26,8 @@ async void Cancel_Button_Clicked(object sender, EventArgs e)
 
         async void Add_Button_Clicked(object sender, EventArgs e)
         {
-            if (string.IsNullOrWhiteSpace(entryObjectName.Text) && string.IsNullOrWhiteSpace(entryObjectCity.Text) &&
-                string.IsNullOrWhiteSpace(entryObjectStreetAndNumber.Text) && string.IsNullOrWhiteSpace(entryObjectPostalCode.Text))
+            if (string.IsNullOrEmpty(entryObjectName.Text) && string.IsNullOrEmpty(entryObjectCountry.Text) && string.IsNullOrEmpty(entryObjectCity.Text) &&
```

Comment:
O kuo čia skiriasi IsNullOrEmpty nuo IsNullOrWhiteSpace?

Reply:
Pasirodo IsNullOrWhiteSpace apsaugo nuo tuščių tarpū, tai vadinasi buvo gerai :D atkeisiu šitą